build-stack.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Script for building Kythe index of Haskell packages.
+# Usage: bash build-stack.sh package...
+
+fail() {
+  (>&2 echo "$1")
+  exit 1
+}
+
+# Directory where to build the index.
+export OUT=/tmp/logs
+
+# REALGHC is used by fake-stack ghc. Note that it must be set before altering
+# the PATH.
+export REALGHC=$(stack path --compiler-exe)
+# Put ghc_kythe_wrapper, ghc-pkg and fake-stack ghc script on the PATH. Note
+# that fake-stack ghc script replaces the system ghc.
+PATH=$PWD/wrappers/stack-docker/fake-stack:$(stack path --compiler-bin):$PATH:$(stack path --local-install-root)/bin
```

Comment:
Could we just modify the PATH locally for the stack command? Like

    PATH=..... stack --system-ghc build ....